src/backend/opencl/platform.cpp

```diff
@@ -37,16 +37,12 @@
 #include <map>
 #include <mutex>
 #include <sstream>
-#include <stdexcept>
 #include <string>
 #include <thread>
 #include <utility>
 #include <vector>
 
-using std::ostringstream;
-using std::runtime_error;
-using std::string;
-using std::vector;
+using namespace std;
```

Comment:
Never use `using namespace std`. Always define each symbol you want to introduce into the file. Even if it is a cpp file.